lib/tasks/ckeditor.rake

```diff
@@ -1,13 +1,26 @@
 require 'fileutils'
 
-desc "Create nondigest versions of all ckeditor digest assets"
-task "assets:precompile" do
-  fingerprint = /\-[0-9a-f]{32}\./
-  for file in Dir["public/assets/ckeditor/**/*"]
-    next unless file =~ fingerprint
-    nondigest = file.sub fingerprint, '.'
-    if !File.exist?(nondigest) or File.mtime(file) > File.mtime(nondigest)
-      FileUtils.cp file, nondigest, verbose: true, preserve: true
+namespace "ckeditor" do
+  desc "Create nondigest versions of all ckeditor digest assets"
+  task "nondigest" => [:environment] do
+    fingerprint = /\-[0-9a-f]{32}\./
```

Comment:
Seems like in the master branch they currently only use `{64}`:
https://github.com/galetahub/ckeditor/blob/82cb23e6872a49bf1b731c0cc3d2c8ae445c7506/lib/tasks/ckeditor.rake#L5